src/assets/scss/modules/_footer.scss

```diff
@@ -1,11 +1,10 @@
 .footer-container {
```

Comment:
@JPOak do you think we should add a `.footer` selector here and just make the `background-color` transparent or white by default? The background color is the whole reason we were opting for option 2 after all.